src/ert/config/commons.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass
+from datetime import datetime
+from pathlib import Path
+from typing import Any, List, Sequence, Tuple
+
+import numpy as np
+import numpy.typing as npt
+
+
+class CustomDict(dict):  # type: ignore
```

Comment:
Nitpick, but could this be moved back? Feel free to move it again later, but preferably in a separate commit, makes it easier to read now.

Reply:
Also applies to Refcase

Reply:
Seems to work so yep, it is now used from `ResponseConfig`, which will then imports it from `ParameterConfig`

Reply:
`Refcase` is a bit trickier, `EnsembleConfig` imports `ResponseConfig`, and `ResponseConfig` now also imports `Refcase` due to the observation parsing being there, so it would be circular if `Refcase` was moved to `EnsembleConfig`